docs/sources/mimir/configure/configure-shuffle-sharding/index.md

```diff
@@ -32,27 +32,27 @@ Configuring a shard size value higher than `0` enables shuffle sharding. The goa
 Shuffle sharding is a technique that isolates different tenant's workloads and gives each tenant a single-tenant experience, even if they're running in a shared cluster.
 For more information about how AWS describes shuffle sharding, refer to [What is shuffle sharding?](https://aws.amazon.com/builders-library/workload-isolation-using-shuffle-sharding/).
 
-Shuffle sharding assigns each tenant a shard that is composed of a subset of the Grafana Mimir instances.
+Shuffle sharding assigns each tenant a shard that is composed of a subset of the Grafana Mimir instances or a subset of Kafka partitions.
 This technique minimizes the number of overlapping instances between two tenants.
 Shuffle sharding provides the following benefits:
 
-- An outage on some Grafana Mimir cluster instances or nodes only affect a subset of tenants.
-- A misbehaving tenant only affects its shard instances.
-  Assuming that each tenant shard is relatively small compared to the total number of instances in the cluster, it’s likely that any other tenant runs on different instances or that only a subset of instances match the affected instances.
+- An outage on some Grafana Mimir cluster instances, Kafka partitions, or nodes only affects a subset of tenants.
+- A misbehaving tenant only affects its shard instances or partitions.
+  Assuming that each tenant shard is relatively small compared to the total number of instances or partitions in the cluster, it’s likely that any other tenant runs on different instances or partitions, or that only a subset of instances match the affected instances.
 
-Using shuffle sharding doesn’t require more resources, but can result in unbalanced instances.
+Using shuffle sharding doesn’t require more resources, but can result in unbalanced instances or partitions.
 
-### Low overlapping instances probability
+### Low overlapping instances or partition probability
 
-For example, in a Grafana Mimir cluster that runs 50 ingesters and assigns each tenant four out of 50 ingesters, by shuffling instances between each tenant, there are 230,000 possible combinations.
+For example, in a Grafana Mimir cluster that runs 50 Kafka partitions and assigns each tenant four out of 50 partitions, shuffling partitions between each tenant creates 230,000 possible combinations.
 
 Randomly picking two tenants yields the following probabilities:
 
-- 71% chance that they do not share any instance
-- 26% chance that they share only 1 instance
-- 2.7% chance that they share 2 instances
-- 0.08% chance that they share 3 instances
-- 0.0004% chance that their instances fully overlap
+- 71% chance that they don't share any partitions
```

Comment:
Do we need to specify instances or partitions here so that we speak to both architectures?